### DISCUSSION TITLE
New user, getting "Error: 403 Forbidden" at login

### DISCUSSION THREAD

**Discussion by chrisisbd** [Kozea/Radicale#1556] [Q&A server]
I have installed 3.0.6 on a Debian 11 virtual server.  I am starting it up using the default systemd configuration that comes with it on Debian.  The server is running OK but on trying to login one always gets "Error: 403 Forbidden" and the login prompt returns. It looks like a permission problem of some sort but I can't find any sort of error log to diagnose it.

How can I get radicale to output errors to somewhere that I can see them?

**Comment by pbiering** [comment]
if started by _systemd_ then it's journal should report messages.

Please check hints in https://github.com/Kozea/Radicale/wiki/Server-Diagnostics---Troubleshooting

Also enable debug level and potentially dedicated debug options

**Comment by chrisisbd** [comment]
On Sun, Aug 11, 2024 at 11:07:54AM -0700, Peter Bieringer wrote:
>    if started by systemd then it's journal should report messages.
> 
>    Please check hints in
>    [1]https://github.com/Kozea/Radicale/wiki/Server-Diagnostics---Troubleshooting
> 
Ah, thank you, that's what I needed to get me going.  All working well
now. :-)

-- 
Chris Green
